vars/getJenkinsLabels.groovy

```diff
@@ -12,8 +12,8 @@ def call(String backend, String aws_region=null) {
     def jenkins_labels = ['aws-eu-west-1': 'aws-sct-builders-eu-west-1',
                           'aws-eu-west-2': 'aws-sct-builders-eu-west-2',
                           'aws-eu-north-1': 'aws-sct-builders-eu-north-1',
-                          'aws-eu-central-1': 'aws-sct-builders-eu-central-1',
-                          'aws-us-east-1' : 'aws-sct-builders-us-east-1',
+                          'aws-eu-central-1': 'aws-sct-builders-eu-central-1-new',
```

Comment:
should change `aws-sct-builders-eu-west-1` -> `aws-sct-builders-eu-west-1-new`

Reply:
and not eu-central-1....

Reply:
anyway, in the EC2 console i don't see the new builder on eu-west-1

Reply:
aws-eu-west-1-qa-builder-9